Documentation/developer-setup-windows.md

```diff
@@ -123,22 +132,21 @@ Then, change into the `C:\Dev\cesium-unreal-samples\Plugins\cesium-unreal\extern
 
 You can also build and install the debug version by using `Debug` or `RelWithDebInfo` instead of `Release`.
 
-# Creating the project files for the Unreal Engine game/project and the Cesium for Unreal plugin
+# Create the VS project files for cesium-unreal-samples
```

Comment:
Unabbreviate "VS" here. This is an extreme nitpick but I think "Creating" matches the titles of the other sections.

```suggestion
# Creating the Visual Studio project files for cesium-unreal-samples
```